jackson-simple/src/test/java/com/baeldung/jackson/annotation/JacksonAnnotationUnitTest.java

```diff
@@ -118,6 +125,13 @@ public void whenDeserializingUsingJsonCreator_thenCorrect() throws IOException {
         assertEquals("My bean", bean.name);
     }
 
+    @Test
```

Comment:
Can we move this method just below the `whenSerializingUsingJsonValueAnnotatedField_thenCorrect`